<?xml version="1.0"?>

<launch>

    <arg name="veh" default="wamv2" />
    <arg name="sync_freq" default="10" />
    <!-- <arg name="real_pose" default="/gazebo/wamv/pose" /> -->
    <arg name="real_pose" default="/wamv/localization_gps_imu/pose" />
    <arg name="sim_pose" default="/gazebo/wamv2/pose" />

    <!--scale 10 times when real is jackal sim is wamv-->
    <arg name="scale_transform" default= "1.0"/>

   <group ns="$(arg veh)">
        <node pkg="vrx_gazebo" type="real2sim_pose_2ws.py" name="real2sim_pose_2ws" output="screen">
            <remap from="joy" to="/joy" />
            <remap from="real_pose" to="$(arg real_pose)" />
            <remap from="sim_pose" to="$(arg sim_pose)" />
            <remap from="real2sim_pose" to="/fake/$(arg veh)"/>
            <param name="sync_freq" value="$(arg sync_freq)" />
            <param name="scale_transform" value="$(arg scale_transform)"/>
            <param name="veh" value="$(arg veh)" />
        </node>
    </group>
</launch>
